app/components/provider_interface/select_course_component.rb

```diff
@@ -0,0 +1,14 @@
+module ProviderInterface
+  class SelectCourseComponent < ViewComponent::Base
+    include ViewHelper
+
+    attr_reader :form_object, :form_path, :page_title, :courses, :course_id
```

Comment:
Is the course_id attr_reader used anywhere?

Reply:
It's used https://github.com/DFE-Digital/apply-for-teacher-training/pull/4241/files#diff-58c92cf3b3dacb6de8b03a18d15bef170341967100187863c5bbfdbd4944e793R4 to set and read the selected course.

Reply:
Ah is that not done inside the form_object (with the attr_accessor)?

Reply:
Ah you are right. We'll remove